engine/common/src/main/java/org/enso/common/DebugServerInfo.java

```diff
@@ -1,8 +1,12 @@
-package org.enso.polyglot.debugger;
+package org.enso.common;
 
-/** Container for Runtime Server related constants. */
+/** Container for debug server related constants. */
 public class DebugServerInfo {
+  private DebugServerInfo() {
+
+  }
   public static final String URI = "enso://debug-server";
   public static final String INSTRUMENT_NAME = "enso-debug-server";
   public static final String ENABLE_OPTION = INSTRUMENT_NAME + ".enable";
+  public static final String FN_OPTION = INSTRUMENT_NAME + ".fn";
```

Comment:
Can we have some documentation on what this option does? I can understand it from context looking at this relatively small PR, but if I were just browsing the whole codebase I would have a very hard time understanding it. Just a short description to correlate it with what it does.

The name is very cryptic - `fn` can mean anything really. Maybe we could use a bit more meaningful name? I don't think length is a big concern here. Could something like `check-warnings-after-function` maybe work?

Reply:
Curious: Why haven't you add javadoc to the other constants [when you created](https://github.com/enso-org/enso/commit/af8b5f88cf7261b3110064a58246b7cae656f569#diff-81e894139b1559eba39e4d3f833d7c1da6d71b7bc6978543ba1caee762021566) the `DebugServerInfo` class?

Reply:
> Consistency check: Why haven't you add javadoc to the other constants [when you created](https://github.com/enso-org/enso/commit/af8b5f88cf7261b3110064a58246b7cae656f569#diff-81e894139b1559eba39e4d3f833d7c1da6d71b7bc6978543ba1caee762021566) the `DebugServerInfo` class?

Because all of these 3 options seemed completely self-explanatory. 

What is the purpose of an `enable` option? That is rather clear.

However, the purpose of an option called `fn` is completely obscure to me. If the option is renamed to something more self-explanatory then perhaps a documentation comment may be unnecessary.

Reply:
Renamed to _method breakpoint_ in 6987125